azure-pipelines.yml

```diff
@@ -41,19 +41,14 @@ jobs:
     path: vscode-website/dist
     persistCredentials: true
   - task: NodeTool@0
-    displayName: Use Node 14.19.3
+    displayName: Use Node 18.x
     inputs:
-      versionSpec: 14.19.3
+      versionSpec: 18.x
```

Comment:
Node 18 is end of life soonish, any reason to not bump to the latest Node 22?

Reply:
The website is still on 18.x